src/main/update_installer.js

```diff
@@ -0,0 +1,43 @@
+
+import { dialog, shell } from "electron";
```

Comment:
Please remove the spaces inside `{` and `}` to match the rest of the repository.